geode-docs/tools_modules/gfsh/command-pages/import.html.md.erb

```diff
@@ -23,79 +23,103 @@ limitations under the License.
+**Import Cluster-Configuration Parameters:**
 
-<span class="tablecap">Table 1. Import Data Parameters</span>
+| Name                                                  | Description                                                                                     |
+|-------------------------------------------------------|-------------------------------------------------------------------------------------------------|
+| <span class="keyword parmname">\\-\\-zip-file-name</span> | *Required.* Name of the zip file containing the cluster configuration artifacts to be imported. |
 
 **Example Commands:**
 
 ``` pre
-import data --region=region2 --file=/mnt5/region2_20121001.gfd --member=server1
+gfsh>import cluster-configuration --zip-file-name=/home/username/myClusterConfig.zip
 ```
 
-## <a id="topic_vnv_grz_ck" class="no-quick-link"></a>import cluster-configuration
+**Sample Output:**
+
+``` pre
+gfsh>import cluster-configuration --zip-file-name=/home/username/myClusterConfig.zip
+Cluster configuration successfully imported
+```
 
-Import a cluster configuration.
+## <a id="topic_jw2_2ld_2l" class="no-quick-link"></a>import data
 
-Imports all artifacts of a cluster configuration from a zip file.
+Import user data from a file or files to a region.
 
 **Availability:** Online. You must be connected in `gfsh` to a JMX Manager member to use this command.
 
-**Note:**
-Shutdown all data members (servers) before importing a shared configuration.
-
 **Syntax:**
 
+If the data was exported sequentially to a single file, use the serial form of the `import` command to import the data:
+
 ``` pre
-import cluster-configuration --zip-file-name=value
+import data --region=value --file=value --member=value [--invoke-callbacks=value]
 ```
 
-<a id="topic_vnv_grz_ck__table_w1m_kfg_2w"></a>
+In this scenario, data from replicated and partitioned regions is imported sequentially from a single file on the specified member.
 
-| Name                                                  | Description                                                                                     |
-|-------------------------------------------------------|-------------------------------------------------------------------------------------------------|
-| <span class="keyword parmname">\\-\\-zip-file-name</span> | *Required.* Name of the zip file containing the cluster configuration artifacts to be imported. |
+See [export data](export.html#topic_263B70069BFC4A7185F86B3272011734) for an explanation of the difference between serial and parallel data exports.
 
-<span class="tablecap">Table 2. Import Cluster-Configuration Parameters</span>
-
-**Example Commands:**
+If the data was exported from a partitioned region in parallel format, use the `--parallel` option to import the data:
 
 ``` pre
-gfsh>import cluster-configuration --zip-file-name=/home/username/myClusterConfig.zip
+import data --parallel --region=value --member=value --dir=value [--invoke-callbacks=value]
```

Comment:
Please inject a newline into the syntax so that there is no need for a horizontal scroll bar in the output.